pkg/yamlmeta/parser.go

```diff
@@ -97,8 +97,8 @@ func (p *Parser) parseBytes(data []byte, lineCorrection int) (*DocumentSet, erro
 			}
 			return nil, err
 		}
-
 		doc := &Document{
+			RawData:  data,
```

Comment:
data hold entire file afaik in this case, seems a bit weird to store file contents on a document.

Reply:
Agreed. The "source" now lives on Position, not the Document.